MovieMVVM/MovieMVVMTests/FileManagerServiceTests.swift

```diff
@@ -0,0 +1,37 @@
+// FileManagerServiceTests.swift
+// Copyright © KarpovaAV. All rights reserved.
+
+@testable import MovieMVVM
+import XCTest
+
+/// Тест сервиса сохранения изображений
+final class FileManagerServiceTests: XCTestCase {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let barText = "bar"
+        static let intOneNumber = 1
+    }
+
+    // MARK: - Private Properties
+
+    private let safeData = Data(capacity: Constants.intOneNumber)
+    private var fileManagerService: FileManagerServiceProtocol?
+
+    // MARK: - Public Methods
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
Надо super добавлять